client/verta/verta/client.py

```diff
@@ -627,7 +654,7 @@ def set_dataset(self, name=None, type="local",
 
         If an accessible Dataset with name `name` does not already exist, it will be created
         and initialized with specified metadata parameters. If such a Dataset does already exist,
-        it will be retrieved; specifying metadata parameters in this case will raise an exception.
+        it will be retrieved; specifying metadata parameters in this case will raise a warning.
```

Comment:
Hmm where is this enforced? I can't seem to find the use of check_unnecessary_params_warning for set_dataset.

Reply:
It was there before this PR here: https://github.com/VertaAI/modeldb/blob/master/client/verta/verta/_dataset.py#L60

Reply:
Oh I missed that. Nice.